pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -2061,7 +2061,6 @@ func (a *ApplicationBackupController) backupResources(
 		if err != nil {
 			return err
 		}
-		return nil
 	}
```

Comment:
One more thing we should take care. I remember seeing this version mismatch error while testing large resource, hence added that return since we will work on the latest AB CR version. Now if we don't return from update then the subsequent prepareResource() will work on stale copy right.. I understand today that call is empty but it may not work when it is populated for some reason in future

Reply:
As posted in the description, update returns the updated copy of the CR, So we will be working on the latest copy of the CR.
```
time="2024-08-26T07:37:52Z" level=info msg="sivakumar --- before CR content &{TypeMeta:{Kind:ApplicationBackup APIVersion:stork.libopenstorage.org/v1alpha1} ObjectMeta:{Name:xxxx-1-67d0014 GenerateName: Namespace:siva-1 SelfLink: UID:e803bbd6-3ceb-4797-81c3-9e8d34f8a363 ResourceVersion:1697227 Generation:6 


time="2024-08-26T07:37:52Z" level=info msg="sivakumar --- after CR content &{TypeMeta:{Kind:ApplicationBackup APIVersion:stork.libopenstorage.org/v1alpha1} ObjectMeta:{Name:xxxx-1-67d0014 GenerateName: Namespace:siva-1 SelfLink: UID:e803bbd6-3ceb-4797-81c3-9e8d34f8a363 ResourceVersion:1697228 Generation:7 

```